pkg/scheduler/event_handler.go

```diff
@@ -131,14 +154,7 @@ func (s *Scheduler) requeueClusterResourceBindings(selector labels.Selector) err
 	}
 
 	for _, clusterResourceBinding := range referenceClusterResourceBindings {
-		key, err := cache.MetaNamespaceKeyFunc(clusterResourceBinding)
-		if err != nil {
-			klog.Errorf("couldn't get key for ClusterResourceBinding(%s): %v", clusterResourceBinding.Name, err)
-			continue
-		}
-		klog.Infof("Requeue ClusterResourceBinding(%s) as placement changed.", clusterResourceBinding.Name)
-		s.queue.Add(key)
-		metrics.CountSchedulerBindings(metrics.PolicyChanged)
```

Comment:
The `requeueClusterResourceBindings` is called when:
1. Policy changed  `onClusterPropagationPolicyUpdate`
2. Cluster changed `enqueueAffectedClusterPolicy`

So, the metric is inaccurate, we should fix it before adding a unit test for it because the code would change accordingly.

Reply:
Done.